source/configuration_manual/acl.rst

```diff
@@ -72,7 +72,7 @@ Sample Configuration:
 Master users
 ============
 
-Master users have their own ACLs. They're not the the mailbox owners, so by
+Master users have their own ACLs. They're not the mailbox owners, so by
```

Comment:
the the changes missing:

Error: source/pigeonhole-man/sievec.1.rst.in:60:22 ... 31, Error - ` the the ` matches a line_forbidden.patterns entry: `\s([A-Z]{3,}|[A-Z][a-z]{2,}|[a-z]{3,})\s\g{-1}\s`. (forbidden-pattern)

Reply:
Thanks, fixed in: https://github.com/dovecot/documentation/compare/731b2a8477713348594d80b71540e5462b8a3093..b1dfaae5a4908d0f11a93addbb30844977ce4348